http://bugs.gentoo.org/166782

2007-03-16  Mike Frysinger  <vapier@gentoo.org>

	* trad-core.c (NBPG): If not defined, set to getpagesize().

--- bfd/trad-core.c
+++ bfd/trad-core.c
@@ -47,6 +47,10 @@ Foundation, Inc., 51 Franklin Street - F
 #include TRAD_HEADER
 #endif
 
+#ifndef NBPG
+# define NBPG getpagesize()
+#endif
+
 struct trad_core_struct
 {
   asection *data_section;
